plugins/woocommerce/src/Admin/Marketing/InstalledExtensions.php

```diff
@@ -380,6 +385,33 @@ protected static function get_mailpoet_extension_data() {
 		return $data;
 	}
 
+	/**
+	 * Get Klaviyo extension data.
+	 *
+	 * @return array|bool
+	 */
+	protected static function get_klaviyo_extension_data() {
+		$slug = 'klaviyo';
+
+		if ( ! PluginsHelper::is_plugin_installed( $slug ) ) {
+			return false;
+		}
+
+		$data         = self::get_extension_base_data( $slug );
+		$data['icon'] = WC_ADMIN_IMAGES_FOLDER_URL . '/marketing/klaviyo.png';
```

Comment:
While the `WC_ADMIN_IMAGES_FOLDER_URL` constant is used elsewhere in this file, it's also marked as `@deprecated`. It might be better to update this reference.

```suggestion
		$data['icon'] = plugins_url( 'assets/images/marketing/klaviyo.png', WC_PLUGIN_FILE );
```

Reply:
Thanks @JPry ! 😀 

For references:

- https://github.com/woocommerce/woocommerce/blob/8978f2a92956072f5c58a153333d9521b49e40a4/plugins/woocommerce/src/Internal/Admin/FeaturePlugin.php#L112-L126 - where `WC_ADMIN_IMAGES_FOLDER_URL` is defined and marked as deprecated.
- https://developer.wordpress.org/reference/functions/plugins_url/ - Retrieves a URL within the plugins or mu-plugins directory.
- https://github.com/woocommerce/woocommerce/blob/8978f2a92956072f5c58a153333d9521b49e40a4/plugins/woocommerce/woocommerce.php#L19-L21 - where `WC_PLUGIN_FILE` is defined.